docs/automations/index.md

```diff
@@ -0,0 +1,23 @@
+---
+icon: kolena/rocket-16
+hide:
+  - toc
+---
+
+# :kolena-rocket-20: Advanced Usage
+
+This section contains tutorial documentation for Kolena automations.
+
+<div class="grid cards" markdown>
+- [:kolena-properties-16: Automatically Extract Text Properties](./extract-text-metadata.md)
```

Comment:
These should use the exact same titles on the sidebar — "Extracting Metadata from Text Fields" and "Setting up Natural Language Search"